dev/tasks/exec/tasks.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const gulp = require( 'gulp' );
+const gutil = require( 'gulp-util' );
+const minimist = require( 'minimist' );
+
+const exec = require( './tasks/exec' );
+const log = require( '../../lib/log' );
+
+module.exports = ( config ) => {
+	const ckeditor5Path = process.cwd();
+	const packageJSON = require( '../../../package.json' );
+
+	// Configure logging.
+	log.configure(
```

Comment:
I think this code isn't needed here... it'd be executed also by some other tasks, what means that it perhaps should be moved to `gulpfile.js` to be executed once or to `lib/log`. But please check if tests still pass.